cmd/kubeadm/app/util/arguments.go

```diff
@@ -65,53 +61,3 @@ func ArgumentsToCommand(base []kubeadmapi.Arg, overrides []kubeadmapi.Arg) []str
 	return command
 }
 
-// ArgumentsFromCommand parses a CLI command in the form "--foo=bar" to an Arg slice
-func ArgumentsFromCommand(command []string) []kubeadmapi.Arg {
```

Comment:
we are going to need this function in a couple of releases when the flag --pod-infra-container is removed from /var/lib/kubelet/kubeadm-flags.env

instead of removing this function could you please add a comment:
```
// ArgumentsFromCommand parses a CLI command in the form "--foo=bar" to an Arg slice.
// This function's primary purpose is to parse the kubeadm-flags.env file, but can remain unused
// for some releases.
```

cc @SataQiu any objections to this approach?

Reply:
@neolit123 I agree with you, `ArgumentsFromCommand` is useful in some cases.
There's no harm in keeping it there.